www/index.js

```diff
@@ -152,16 +159,16 @@ HotPush.prototype._hasloadedLocalFile = function() {
 HotPush.prototype.update = function() {
   var self = this;
   if (this.options.type === 'replace') {
-    this._syncs = [ContentSync.sync({ src: this.options.archiveURL, id: 'assets', headers: this.options.headers})];
+    this._syncs = [ContentSync.sync({ src: this.options.archiveURL, id: 'assets', copyCordovaAssets: true, headers: this.options.headers})];
```

Comment:
Ok, I've only tested it on Android. In our app react bundled files are loaded into the body not head.